src/functionalTest/java/uk/gov/hmcts/reform/ccd/document/am/befta/DataStoreTestAutomationAdapter.java

```diff
@@ -0,0 +1,126 @@
+package uk.gov.hmcts.reform.ccd.document.am.befta;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.hmcts.befta.BeftaMain;
+import uk.gov.hmcts.befta.DefaultTestAutomationAdapter;
+import uk.gov.hmcts.befta.data.UserData;
+import uk.gov.hmcts.befta.exception.FunctionalTestException;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DataStoreTestAutomationAdapter extends DefaultTestAutomationAdapter {
+
+    private Logger logger = LoggerFactory.getLogger(DataStoreTestAutomationAdapter.class);
+
+    private static final String[] TEST_DEFINITIONS_NEEDED_FOR_TA = {
+        "src/functionalTest/resources/CCD_CNP_27.xlsx",
+        "src/functionalTest/resources/CCD_CNP_27_AUTOTEST1.xlsx",
+        "src/functionalTest/resources/CCD_CNP_27_AUTOTEST2.xlsx",
+        "src/functionalTest/resources/CCD_CNP_RDM5118.xlsx",
+
+        "src/functionalTest/resources/CCD_BEFTA_JURISDICTION1.xlsx",
```

Comment:
Please remove loading of all excel files except CCD_BEFTA_JURISDICTION2. Kindly delete corresponding files as well.

Reply:
Removed other excel files.